components/Item/UpNext.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <v-fade-transition>
+    <v-card class="container pa-4">
+      <div class="d-flex flex-column flex-grow-1">
+        <v-card-title class="countdown-header pa-0">
+          {{ $t('dialog.upNext.nextEpisodePlayingIn') }}
+          <span class="primary--text darken-2">
+            &ensp;{{ timeLeft }} {{ $t('seconds') }}
+          </span>
+        </v-card-title>
+        <v-card-subtitle class="title subtitle-1 pa-0">
+          {{ nextSeriesName }} -
+          {{
+            $t('tvShowAbbrev', {
+              seasonNumber: nextSeasonNumber,
+              episodeNumber: nextEpisodeNumber
+            })
+          }}
+          - <span v-if="$vuetify.breakpoint.xsOnly"> <br /> </span>
+          {{ nextName }}
+        </v-card-subtitle>
+        <div>
+          {{ nextRunTime }}
+          <span class="pl-4">{{ $t('endsAt', { time: nextEndsAt }) }} </span>
+        </div>
+        <v-card-actions>
+          <v-spacer />
+          <v-btn class="primary darken-2" @click="$emit('startNext')">
+            {{ $t('dialog.upNext.startNow') }}
+          </v-btn>
+          <v-btn @click="$emit('hide')"> {{ $t('dialog.upNext.hide') }}</v-btn>
+        </v-card-actions>
+      </div>
+    </v-card>
+  </v-fade-transition>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { mapState, mapGetters, mapActions } from 'vuex';
+import timeUtils from '~/mixins/timeUtils';
+
+export default Vue.extend({
+  mixins: [timeUtils],
+  props: {
+    timeLeft: {
+      type: Number,
+      required: true
+    }
+  },
+  data() {
+    return {
+      source: ''
+    };
+  },
+  computed: {
+    ...mapState('playbackManager', ['queue', 'currentItemIndex']),
+    ...mapGetters('playbackManager', ['getNextItem']),
+    nextSeriesName(): string {
+      return this.getNextItem?.SeriesName;
+    },
+    nextEpisodeNumber(): number {
+      return this.getNextItem?.IndexNumber;
+    },
+    nextSeasonNumber(): number {
+      return this.getNextItem?.ParentIndexNumber;
+    },
+    nextRunTime(): string {
+      return this.getRuntimeTime(this.getNextItem?.RunTimeTicks);
+    },
+    nextEndsAt(): string {
+      const seconds =
+        this.ticksToMs(this.getNextItem?.RunTimeTicks) + this.timeLeft * 1000;
+      return this.$dateFns.format(Date.now() + seconds, 'p');
+    },
+    nextName(): string {
+      return this.getNextItem?.Name;
+    }
```

Comment:
This entire section is a bit unclean, imo.

It'd be better (and probably a bit more optimized, since each compute creates a new reactive thing) to just use getNextItem with the property right after it.